XenAdmin/Dialogs/PropertiesDialog.cs

```diff
@@ -202,6 +203,9 @@ private void Build()
                 if (is_pool_or_standalone && !Helpers.FeatureForbidden(xenObject.Connection, Host.RestrictLivePatching))
                     ShowTab(LivePatchingEditPage = new LivePatchingEditPage());
 
+                if (is_pool_or_standalone && !Helpers.FeatureForbidden(xenObject.Connection, Host.RestrictIGMPSnooping) && Helpers.GetMaster(pool).vSwitchNetworkBackend)

```

Comment:
`vSwitchNetworkBackend` is now a method, so this needs to be changed to `vSwitchNetworkBackend()`

Reply:
Updated. Thanks.